piwigo-random-backend.php

```diff
@@ -86,14 +93,27 @@ function check_param($param, $values) {
   exit(1);
 }
 
-$url = $site . "ws.php" .
-  "?format=php" .
-  "&method=pwg.categories.getImages" .
-  ($cat_id ? "&cat_id=" . $cat_id : "") .
-  "&recursive=true" .
-  "&per_page=" . $maximages .
-  "&page=1" .
-  "&order=random";
+if ($tag_name)
```

Comment:
This should be better factored. Most of the code is different in the "then" and "else" branch and should be moved out of the if/else.

Reply:
I find it more readable to have the entire url construction in the if/else, but ok, will do.